martian/src/mro.rs

```diff
@@ -478,7 +478,7 @@ pub enum Volatile {
 
 impl Default for Volatile {
     fn default() -> Self {
-        Volatile::Strict
+        Volatile::False
```

Comment:
Do not change this.  `Volatile::False` should definitely not be the default - that would be an extreme change relative to the previous behavior (which was `None`).

Reply:
I can revert this change if you prefer. As far as I can tell it would have to material change to the intention of this PR, since the above change changes the default to `None`.